src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import './Navbar.css';
```

Comment:
I highly recommend using [CSS modules](https://css-tricks.com/css-modules-part-1-need/) to scope your CSS locally to the component you wrote it for, and using the `index.css` file for your global CSS.

To minimize any CSS conflicts or breaking changes that would occur as Talawa Admin grows. This is primarily due to CSS being scoped globally in react, and CSS modules being one solution to scoping your css to your components locally

Reply:
sure sir changing it now